pontoon/messaging/static/css/email_consent.css

```diff
@@ -65,10 +69,13 @@ body > header.menu-opened {
   font-size: 22px;
   font-weight: 300;
   line-height: 36px;
-  margin-bottom: 60px;
   width: 900px;
 }
 
+#main p.main-text {
+  margin-bottom: 60px;
+}
+
```

Comment:
Changes to this file drop the scrollbar on the Email consent page, so they are indirectly related to the issue the patch is fixing.